gradle.properties

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-release_version=3.39.2
+release_version=3.39.3
```

Comment:
I think we should update the minor version because the minor version of the library is changed